include/constants/moves.h

```diff
@@ -476,6 +476,19 @@
 #define CLASS_SPECIAL  1
 #define CLASS_STATUS   2
 
+#define RANGE_0                          0
```

Comment:
spacing is slightly off

Reply:
it's meant to be like that (lines up with the numbers rather than the parenthesis)